### PR TITLE
[Cloud-centric] Self-hosted backup and restore

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Edited self-hosted backup and restore pages for cloud-centric updates

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: self-hosted/backup-and-restore/pg-dump-and-restore.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ the native PostgreSQL [`pg_dump`][pg_dump] and [`pg_restore`][pg_restore]
...
 don't need to backup and restore your data. See
 the [upgrading instructions][timescaledb-upgrade].
 
 <Highlight type="warning">
 If you are using this `pg_dump` backup method regularly, make sure you keep
-track of which versions of PostgreSQL and TimescaleDB you are running. For more
+track of which versions of PostgreSQL and Timescale you are running. For more
 information, see "Versions are mismatched when dumping and restoring a database"
-in the [Troubleshooting section](https://docs.timescale.com/timescaledb/latest/how-to-guides/backup-and-restore/troubleshooting/).
+in the [Troubleshooting section](https://docs.timescale.com/timescaledb/latest/self-hosted/backup-and-restore/troubleshooting/).
```
Comment (Rajakavitha1) [inline self-hosted/backup-and-restore/pg-dump-and-restore.md:24]: Hi @charislam! I am not sure about the URL here. Please help.

File: self-hosted/backup-and-restore/docker-and-wale.md [hunk 2]
```diff
@@ -9,45 +9,45 @@ tags: [restore, recovery, physical backup]
...
 
-When you run TimescaleDB in a containerized environment, you can use
+When you run Timescale in a containerized environment, you can use
 [continuous archiving][pg archiving] with a [WAL-E][wale official] container.
 These containers are sometimes referred to as sidecars, because they run
-alongside the main container. We provide a [WAL-E sidecar image][wale image]
-that works with TimescaleDB as well as regular PostgreSQL. In this section, we
-help you set up archiving to your local filesystem with a main TimescaleDB
+alongside the main container. A [WAL-E sidecar image][wale image]
+that works with Timescale as well as regular PostgreSQL. In this section, you
```
Comment (Loquacity) [inline self-hosted/backup-and-restore/docker-and-wale.md:16]:
```suggestion
works with Timescale as well as regular PostgreSQL. In this section, you 
```

File: self-hosted/backup-and-restore/docker-and-wale.md [hunk 3]
```diff
@@ -9,45 +9,45 @@ tags: [restore, recovery, physical backup]
...
+can set up archiving to your local filesystem with a main Timescale
 container called `timescaledb`, and a WAL-E sidecar called `wale`. When you are
 ready to implement this in your production deployment, you can adapt the
 instructions here to do archiving against cloud providers such as AWS S3, and
 run it in an orchestration framework such as Kubernetes.
 
 <Highlight type="deprecation">
 This section describes using the WAL-E sidecar for archiving, which is deprecated
-on TimescaleDB. We strongly recommend that you do not use this feature in a
+on Timescale. We strongly recommend that you do not use this feature in a
```
Comment (Loquacity) [inline self-hosted/backup-and-restore/docker-and-wale.md:25]: There's a partial for this.

File: self-hosted/backup-and-restore/pg-dump-and-restore.md (hunk 1 above)
Comment (Loquacity) [reply self-hosted/backup-and-restore/pg-dump-and-restore.md:24]: that should be right 👍

**Review by Loquacity — APPROVED** [review]
Couple of small things.

File: self-hosted/backup-and-restore/docker-and-wale.md (hunk 3 above)
Comment (Rajakavitha1) [reply self-hosted/backup-and-restore/docker-and-wale.md]: Thank you @Loquacity !!! just added the partial.

**Comment by charislam** [comment]
Again, looks good but maybe we need to align on branding of self-hosted (is it `Timescale` or `TimescaleDB` or `timescaledb`? PMM seems to be coming down pretty strongly against option 1 but not sure about 2 or 3

**Review by charislam — APPROVED** [review]

**Comment by Loquacity** [comment]
> Again, looks good but maybe we need to align on branding of self-hosted (is it `Timescale` or `TimescaleDB` or `timescaledb`? PMM seems to be coming down pretty strongly against option 1 but not sure about 2 or 3

Let's do that wholescale when we decide how to go forward. I don't think it will be a major change, and I don't want to hold up these efforts unduly.

**Pull request merged**